### ISSUE TITLE
delete vfs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4934]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#4934|
|Type|enhancement|
|Created|2011-03-10 23:07:14Z|
|Branch|https://github.com/twisted/twisted/tree/remove-vfs-4934|

It's old, it's gross, nothing uses it, it's not in the release.

<details><summary>Searchable metadata</summary>

```
trac-id__4934 4934
type__enhancement enhancement
reporter__glyph glyph
priority__high high
milestone__ 
branch__branches_remove_vfs_4934 branches/remove-vfs-4934
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__vfs vfs
keywords__ 
time__1299798434000000 1299798434000000
changetime__1299812844000000 1299812844000000
version__None None
owner__exarkun exarkun
cc__cablehead
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30916]) Branching to 'remove-vfs-4934'

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @glyph<br>@glyph set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun removed owner<br>@exarkun set status to `new`|
|-|-|

I deleted it very much.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @glyph<br>@glyph set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

[Build Results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-vfs-4934)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph set owner to @exarkun<br>@glyph set status to `new`|
|-|-|

You deleted it the most.

Goodbye, vfs, you will not be missed.  (Now: on to `twisted.path`!)

Please land.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [30926]) Merge remove-vfs-4934 

Author: exarkun
Reviewer: glyph
Fixes: [#4934](https://github.com/twisted/twisted/issues/4934)

Remove twisted.vfs, unreleased, unmaintained, incomplete.
